src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs

```diff
@@ -64,6 +64,13 @@ private ProcessArgumentBuilder GetArguments(FilePath solution, MSBuildSettings s
                 builder.Append(string.Concat("/p:\"Configuration\"=", configuration.Quote()));
             }
 
+            // Buid for a specific platform?
+            if (settings.PlatformTarget != default(PlatformTarget))
```

Comment:
Would be better if we made `PlatformTarget` nullable and only set the platform target if it's not null. This would be more in line with how we do it at other places.